### PR TITLE
WPF-60544 New UG documentation for DeleteCommand and DeleteParameter

### PR DESCRIPTION
**Pull request by deepathiruppathy01**
Added UG document for DeleteCommand and DeleteCommand parameters.

### CODE REVIEW

**Comment by deepathiruppathy01** [comment]
rebuild

File: wpf/Diagram/Commands/Delete.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+title: Delete Command in WPF Diagram control | Syncfusion
+description: Learn here all about Delte Command support in Syncfusion WPF Diagram (SfDiagram) control, its elements and more.
+platform: wpf
+control: SfDiagram
+documentation: ug
+---
+
+# Delete Command in WPF Diagram (SfDiagram)
+
+The [Delete](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Diagram.IDiagramCommands.html#Syncfusion_UI_Xaml_Diagram_IDiagramCommands_Delete) commands are used to do delete operation on the Diagram view for currently selected item. This command is also used to do delete any un-selected diagram objects with its parameter.
```
Comment (laksmipriyamurugesan) [inline wpf/Diagram/Commands/Delete.md:12]: This command is also used to delete any un-selected diagram objects with its parameter.

File: wpf/Diagram/Commands/Delete.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+platform: wpf
+control: SfDiagram
+documentation: ug
+---
+
+# Delete Command in WPF Diagram (SfDiagram)
+
+The [Delete](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Diagram.IDiagramCommands.html#Syncfusion_UI_Xaml_Diagram_IDiagramCommands_Delete) commands are used to do delete operation on the Diagram view for currently selected item. This command is also used to do delete any un-selected diagram objects with its parameter. 
+
+To execute delete commands, DeleteCommandParameter type have to be passed.
```
Comment (laksmipriyamurugesan) [inline wpf/Diagram/Commands/Delete.md:14]: To execute delete commands, the DeleteCommandParameter type has to be passed.

File: wpf/Diagram/Commands/Delete.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Delete command parameter
+
+The DeleteParameter is used to represent the items parameters to for executing delete command. Please find its properties and their description as follows. The DeleteParameter contains a `Items` property and it is used to specify a list of diagram objects need to be delete using delete command.
```
Comment (laksmipriyamurugesan) [inline wpf/Diagram/Commands/Delete.md:18]: The DeleteParameter is used to represent the item's parameters for executing the delete command. Please find its properties and their description as follows. The DeleteParameter contains a `Items` property and it is used to specify a list of diagram objects that need to be deleted using the delete command.

File: wpf/Diagram/Commands/Delete.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+platform: wpf
+control: SfDiagram
+documentation: ug
+---
+
+# Delete Command in WPF Diagram (SfDiagram)
+
+The [Delete](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Diagram.IDiagramCommands.html#Syncfusion_UI_Xaml_Diagram_IDiagramCommands_Delete) commands are used to do delete operation on the Diagram view for currently selected item. This command is also used to delete any un-selected diagram objects with its parameter. 
+
+To execute delete commands, the DeleteCommandParameter type has to be passed.
```
Comment (sarathkumarvenkatesan) [inline wpf/Diagram/Commands/Delete.md:14]: use api link for DeleteCommandParameter

**Review by sarathkumarvenkatesan — APPROVED** [review]

**Review by sarathkumarvenkatesan — APPROVED** [review]

**Pull request merged**